kornia/augmentation/augmentation.py

```diff
@@ -671,10 +759,22 @@ def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor
         return rg.random_rotation_generator(batch_shape[0], degrees, self.same_on_batch, self.device, self.dtype)
 
     def compute_transformation(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.compute_rotate_tranformation(input, params)
+        angles: torch.Tensor = params["degrees"].to(input)
```

Comment:
Use get rotation matrix

Reply:
I do not understand what you mean here.

Reply:
```get_rotation_matrix2d``` is not pretty efficient since it will need scale tensor as well. I will keep the current implementation for now. We might update ```get_rotation_matrix2d``` later.

Reply:
add todo note here